scripts/erc20_to_native/sendForeign.js

```diff
@@ -0,0 +1,95 @@
+const path = require('path')
+require('dotenv').config({
+  path: path.join(__dirname, '../../.env')
+})
+const Web3 = require('web3')
+const Web3Utils = require('web3-utils')
+const rpcUrlsManager = require('../../src/services/getRpcUrlsManager')
+const { sendTx, sendRawTx } = require('../../src/tx/sendTx')
+
+const {
+  USER_ADDRESS,
+  USER_ADDRESS_PRIVATE_KEY,
+  FOREIGN_BRIDGE_ADDRESS,
+  FOREIGN_MIN_AMOUNT_PER_TX,
+  ERC20_TOKEN_ADDRESS
```

Comment:
Can `ERC20_TOKEN_ADDRESS` will be extracted from the bridge contract?

Reply:
Done.